source/states/stages/Tank.hx

```diff
@@ -10,65 +10,78 @@ class Tank extends BaseStage
 	var tankWatchtower:BGSprite;
 	var tankGround:BackgroundTank;
 	var tankmanRun:FlxTypedGroup<TankmenBG>;
-	var foregroundSprites:FlxTypedGroup<BGSprite>;
+	var boppers:FlxTypedGroup<BGSprite>;
```

Comment:
you should probably revert this name change because that will break mods that use `foregroundSprites` as a variable in lua/hscript